app/views/users/show.html.erb

```diff
@@ -10,8 +10,11 @@
     <li> <%= current_user.first_name  %> <%= current_user.last_name %> </li>
     <li> <%= current_user.email%> </li>
   </ul>
-  <section>
+  <section class="bookmarked-segments">
     <h1>Bookmarked Segments</h1>
```

Comment:
Wondering if I should hide this, it wasn't specified in the user story so I left it alone